services/brig/src/Brig/App.hs

```diff
@@ -491,7 +491,7 @@ instance Applicative (AppT r) where
   (AppT x0) <*> (AppT x1) = AppT $ x0 <*> x1
 
 instance Monad (AppT r) where
-  (AppT x0) >>= f = AppT $ x0 >>= unAppT . f
+  (AppT x0) >>= f = AppT $ x0 >>= (\x -> unAppT $ f x)
```

Comment:
i don't particularly like this change.  why did it become necessary?

Reply:
GHC2021 disables the `DeepSubsumption` extension by default, so the options were to either enable it for this file or use an explicit lambda like this, or add some eldritch looking type signatures. After some digging and some thought, I think we're fine with either the extension or the eta expansion. I reverted this change and enabled the extension for this file.